evalml/objectives/fraud_cost.py

```diff
@@ -90,7 +73,9 @@ def objective_function(self, y_true, y_predicted, X, sample_weight=None):
         # calculate money lost from fees
         false_positives = (~y_true & y_predicted) * interchange_cost
 
-        loss = false_negatives.sum() + false_positives.sum()
+        # add a penalty if we output naive predictions
+        all_one_prediction_cost = (2 - len(set(y_predicted))) * fraud_cost.sum()
```

Comment:
Add a penalty to naive predictions. Without this penalty, we could incur small fraud costs if we predict every transaction as fraudulent.

Reply:
And this "small fraud cost" is currently not captured by the objective, right? I guess I'm wondering why this isn't currently captured by how the objective is computed.

Reply:
It is currently captured, but that would only be the cost coming from `false_positive` cases, which small, especially when `interchange_cost` is small. I wanted to add a big penalization to prevent us from choosing a naive estimator.